app/views/collection_profile/show.html.erb

```diff
@@ -98,7 +98,11 @@
 
         <% if @collection.challenge %>
           <li><%= link_to ts('Delete Challenge'),
-                          eval("collection_#{challenge_class_name(@collection)}_path(@collection)"),
+                          if  @collection.prompt_meme? then
```

Comment:
This has the same extra space and unnecessary `then` problem as the previous view file, btw.